docs/tutorials/t1.rst

```diff
@@ -102,6 +108,62 @@ using ``child_data``
         for result in sub_data.analysis_results():
             print(result)
 
+:math:`T_1` experiments with kerneled measurement
+---------------------------------------------------
+:math:`T_1` experiment can also be done with kerneled measurement.
+If we will set the running option `meas_level=MeasLevel.KERNELED`, the
```

Comment:
```suggestion
If we set the runn option `meas_level=MeasLevel.KERNELED`, the
```

Reply:
Changed as suggested (without the typo).
```If we set the run option `meas_level=MeasLevel.KERNELED`, the job```